mongo/mongo_test.go

```diff
@@ -116,3 +121,107 @@ func TestRoundTripProcessError(t *testing.T) {
 
 	assert.Equal(t, description.ServerKind(description.Unknown), m.Description().Servers[0].Kind, "Failed to update the server Kind to Unknown")
 }
+
+func TestMongo_RoundTrip_Cursor(t *testing.T) {
+	const uri = "mongodb://127.0.0.1:8001/?loadBalanced=true"
```

Comment:
This will have to be changed to reflect local and CI requirements for MongoBetween development